select2-parent/select2/src/main/java/org/wicketstuff/select2/Settings.java

```diff
@@ -53,7 +55,6 @@
 	private String initSelection;  //TODO this will not work
 	private String query;
 	private String width;
-	private String theme;
```

Comment:
Actually 'templating' from your last comment is supported by 'templateResult' settings (we are using it in our project)
You are creating 'fake' empty theme if only name was provided, my mistake, comment too early
Could you please update spaces and I'll merge the change